cmd/bm/fluent_forward_go/bm_test.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+//go test -benchmem -benchtime=100x -run=^$ -bench ^Benchmark.*$ github.com/IBM/fluent-forward-go/cmd/bm/fluent_forward_go
+
+import (
+	"testing"
+	"time"
+
+	"github.com/IBM/fluent-forward-go/fluent/client"
+	"github.com/IBM/fluent-forward-go/fluent/protocol"
+)
+
+func Benchmark_Fluent_Forward_Go_SendOnly(b *testing.B) {
+	tagVar := "bar"
+
+	c := &client.Client{
+		Timeout: 3 * time.Second,
+		ConnectionFactory: &client.TCPConnectionFactory{
+			Target: client.ServerAddress{
+				Hostname: "localhost",
+				Port:     24224,
+			},
+		},
+	}
```

Comment:
Why not using our own stuff?

```
c = client.New(client.ConnectionOptions{})
c.Timeout = 3 * time.Second
```

If changing here also change the occurrences below